scripts/cnn_feat_extraction/example.cfg

```diff
@@ -2,7 +2,7 @@
 # Model directory
 model_dir = caffenet
 # Image set directory (assumes images are in subdirectories, one per class)
-imageset_dir = data/imageset
+imageset_dir = yellow_imageset_flat
```

Comment:
Did you want to have this set to yellow_imageset_flat in the example.cfg file?

Reply:
Absolutely not, good catch!

Reply:
To fix, you can just push a new commit and it will fold it into this pull request.